### PR TITLE
Docs : Update guide for TIZEN RT IDE.

### PR DESCRIPTION
**Pull request by Hyunjun85**
Users can get and manage the certificate referring this.

1. Update HowToDevelopThingsWithTizenStudioForRT.md.
2. Add manage-certificate.md.
3. Add HowToManageDeviceModel.md.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToManageDeviceModel.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# How to Import / Export device Model.
```
Comment (sunghan-chang) [inline docs/HowToManageDeviceModel.md]: 1. Could you check this file on markdown view? It looks not good. You need to change a line between picture and sentence.
2. This is content of SmartThings. But I can't know SmartThings doc from title.

File: docs/manage-certificate.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+
+# Managing SmartThings&trade; Certificates
+
```
Comment (sunghan-chang) [inline docs/manage-certificate.md]: If you provide contents at start of doc, it makes easy to know and easy to find what and where it is.
```
## Contents
> [Create certi....](#create...)
> [Import xxx](#...)
> [Activate xxx](#...)
```

File: docs/HowToManageDeviceModel.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+   2. In the Add Resource window, select the type of resource to be added, edit its information, and click **OK**.  
+   ![Add Resource](media/rt_model_add_resource_window.png)  
+   The new resource appears in the list in the **Resource/Property** section of the Model Manager main view.  
+- To restore the resources, click **Restore default**. This restores the initial resource state of the device, deleting all changes that have been made.  
+![The imported device shows up](media/rt_model_import_model_finished.png)  
+
+To export a device model:  
+1. Select the model to be exported and click **Export** on the **Device/Platform** toolbar.  
+![Export the model](media/rt_model_export_model.png)  
+2. The device model is exported ad a JSON file. Enter a file name for the exported model and click **OK**.
```
Comment (juitem) [inline docs/HowToManageDeviceModel.md]: ad -> as

File: docs/manage-certificate.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    -  Click **Download** and specify a location where the certificate file is saved.
+    -  In the Certificate Generation Dialog, click **Browse**, select the downloaded certificate file, and click **OK**.
+
+    ![Creating a certificate in the Certificate Generation Dialog](media/rt_cert_create_download.png)
+
+<a name="import"></a>
+## Importing a Certificate
+
+To import an existing certificate:
+1.  In the SmartThings Certificate Manager, click **Import**.
```
Comment (juitem) [inline docs/manage-certificate.md]: count of  spaces after 1. and 2. are different

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**